src/Apps/IdentityVerification/IdentityVerificationApp/index.tsx

```diff
@@ -225,8 +239,15 @@ const IdentityVerificationApp: React.FC<Props> = ({
               <Spacer y={1} />
 
               <Text variant="xs" color="black60" textAlign="center">
-                For more information, see the{" "}
-                <RouterLink to="/identity-verification-faq">FAQ</RouterLink>.
+                Questions about identity verification? Read the{" "}
+                <RouterLink to="https://support.artsy.net/hc/en-us/articles/360062798613-How-to-Complete-Identity-Verification">
```

Comment:
**suggestion:** Since we're adding a new Zendesk link, we should let @sweir27 know about this so it can be added to the list of URLs in Force to be migrated to Salesforce.

Reply:
Done.